weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/SimplePageLayout.tsx

```diff
@@ -41,6 +41,7 @@ export const SimplePageLayout: FC<{
   }>;
   leftSidebar?: ReactNode;
   hideTabsIfSingle?: boolean;
+  headerExtra?: ReactNode;
```

Comment:
adds a prop similar to SimplePageLayoutWithHeader
where we can add extra actions